meta/main.yml

```diff
@@ -1,20 +1,68 @@
 # SPDX-License-Identifier: MIT
 ---
 galaxy_info:
-  author: John Doe <jdoe@corp.com>
-  description: Basic template for Linux system roles
-  company: John Doe, Inc.
+  # Replace with role's author name:
+  author: Joerg Kastning <jkastning@my-it-brain.de>
+  # Replace with the real description of what is role's purpose:
+  description: Install, configure and operate AIDE
+  # Replace with the company the role's author is member of:
+  company: Red Hat
+
+  # If the issue tracker for your role is not on github, uncomment the next
+  # line and provide a value
+  # issue_tracker_url: http://example.com/issue/tracker
+
+  # Some suggested licenses:
+  # - BSD (default)
+  # - MIT
+  # - GPLv2
+  # - GPLv3
+  # - Apache
+  # - CC-BY
   license: MIT
+
   min_ansible_version: "2.9"
+
+  # Optionally specify the branch Galaxy will use when accessing the GitHub
+  # repo for this role. During role install, if no tags are available, Galaxy
+  # will use this branch. During import Galaxy will access files on this
+  # branch. If Travis integration is configured, only notifications for this
+  # branch will be accepted. Otherwise, in all cases, the repo's default branch
+  # (usually main) will be used.
+  # github_branch:
+
+  #
+  # platforms is a list of platforms, and each platform has a name and a list
+  # of versions.
+  #
+  # platforms:
+  # - name: Fedora
+  #   versions:
+  #   - all
+  #   - "25"
+  # - name: SomePlatform
+  #   versions:
+  #   - all
+  #   - "1.0"
+  #   - "7"
+  #   - "99.99"
   platforms:
+    # Replace the below with your platform list:
     - name: Fedora
       versions:
         - all
     - name: EL
       versions:
-        - "9"
-  galaxy_tags:
-    - el9
-    - el10
-    - fedora
+        - all
+
+  galaxy_tags: []
```

Comment:
1. `all` is not a valid specifier under `EL` - you must specify the major versions explicitly - so if `aide` is supported on EL 9 and 10, you must use

```yaml
    - name: EL
      versions:
        - "9"
        - "10"
```

2. We have to use galaxy_tags now to specify platform support - support for `platforms:` is being deprecated - see https://github.com/linux-system-roles/.github/pull/63 for additional information - this is why the template has explicit `galaxy_tags` - so if the `aide` role supports EL 9, 10, and Fedora, you need to have `galaxy_tags` like this:

```yaml
  galaxy_tags:
    - el9
    - el10
    - fedora
```